src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -258,13 +207,11 @@ void givenExistingUsernameFromAdminUserRegisterRequest_whenAdminUserExist_thenTh
         // Then
         Assertions.assertThrows(
                 AysAdminUserAlreadyExistsByUsernameException.class,
-                () -> adminUserRegisterService.register(mockAdminUserRegisterRequest)
+                () -> adminUserRegisterService.completeRegistration(applicationId, mockAdminUserRegisterApplicationCompleteRequest)
         );
 
         Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Ekledim.